setup_go.sh

```diff
@@ -7,20 +7,50 @@ set -ex
 # Note: the official go build uses the fourth option (bootstrap the build using go1.4), but according to the documentation
 # this shouldn't make a difference.
 
+# Upgrade bison 
+curl -sL -O https://ftp.gnu.org/gnu/bison/bison-${BISON_VERSION}.tar.gz
+echo "${BISON_SHA256}  ./bison-${BISON_VERSION}.tar.gz" | sha256sum --check
+tar -zxvf ./bison-${BISON_VERSION}.tar.gz
+cd bison-${BISON_VERSION}
+./configure --prefix=/usr/local/bison && make && make install
+cd -
+
+export PATH=/usr/local/bison/bin:$PATH
+
+# Upgrade binutils
+curl -sL -O https://ftp.gnu.org/gnu/binutils/binutils-${BINUTILS_VERSION}.tar.gz
+echo "${BINUTILS_SHA256}  ./binutils-${BINUTILS_VERSION}.tar.gz" | sha256sum --check
+tar -zxvf ./binutils-${BINUTILS_VERSION}.tar.gz
+cd binutils-${BINUTILS_VERSION}
+./configure --prefix=/usr/local/binutils --disable-gprofng && make && make install
+cd -
+
 # Install gimme to get go1.15.11
-curl -sL -o /bin/gimme https://raw.githubusercontent.com/travis-ci/gimme/v1.5.4/gimme
+curl -sL -o /bin/gimme https://raw.githubusercontent.com/travis-ci/gimme/v1.5.5/gimme
 # Check sha256sum, fail otherwise
-echo "03b295636d4e22870b6f6e9bc06a71d65311ae90d3d48cbc7071f82dd5837fbc  /bin/gimme" | sha256sum --check
+echo "3d565d57ec28edb14be9e540cd3e628607ec5b791e78224c47250d36ce4aedf2  /bin/gimme" | sha256sum --check
 chmod +x /bin/gimme
-eval "$(gimme 1.15.11)"
+eval "$(gimme 1.18.9)"
```

Comment:
Why 1.18.9 and not 1.19.4?

Reply:
It's the bootstrap golang; I am OK with setting it to `1.19.4`, I juste operated under the assumption here that there was a reason why we wanted to bootstrap with a prior version.

Reply:
You're right, feel free to disregard the comment.